drivers/bt_ll_nrfxlib/entropy/entropy_ll_nrfxlib.c

```diff
@@ -126,7 +126,7 @@ static const struct entropy_driver_api rng_driver_api_funcs = {
 	.get_entropy_isr = rng_driver_get_entropy_isr
 };
 
-DEVICE_AND_API_INIT(rng_driver, DT_LABEL(DT_CHOSEN(zephyr_entropy)),
+DEVICE_AND_API_INIT(rng_driver, DT_LABEL(DT_NODELABEL(rng)),
```

Comment:
Not related to this line, but to the commit message:
`drivers: bt_ll_nrfxlib: fix nrg driver device name`, you mean `rng driver device name`

Reply:
I make this typo all the time for some reason. Thanks.